vscode-powershell.build.ps1

```diff
@@ -113,7 +113,7 @@ task Test Build, {
     Write-Host "`n### Running extension tests" -ForegroundColor Green
     Invoke-BuildExec { & npm run test }
     # Reset the state of files modified by tests
-    Invoke-BuildExec { git checkout package.json test/.vscode/settings.json}
```

Comment:
Is this one getting modified by tests too now?

Reply:
I don't think the package.json was getting modified before, it might be cruft, but I didn't want to remove it just in case. It doesn't hurt anything to have it here.

Reply:
There was a while that the VS Code test package was modifying that file and it seemed to be messing stuff up somehow, hence the origin of this. It can probably go away but yeah also it doesn't hurt anything.